moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -676,45 +676,19 @@ double ServoCalcs::velocityScalingFactorForSingularity(const Eigen::VectorXd& co
   return velocity_scale;
 }
 
-void ServoCalcs::enforceSRDFAccelVelLimits(Eigen::ArrayXd& delta_theta)
+void ServoCalcs::enforceVelLimits(Eigen::ArrayXd& delta_theta)
 {
   Eigen::ArrayXd velocity = delta_theta / parameters_.publish_period;
   const Eigen::ArrayXd acceleration = (velocity - prev_joint_velocity_) / parameters_.publish_period;
 
   std::size_t joint_delta_index = 0;
+  // Track the smallest velocity scaling factor required, across all joints
+  double min_relative_velocity_all_joints = 1;
```

Comment:
Your comment is very descriptive - I'd rename the var accordingly to something like `velocity_limit_scaling_factor` or `velocity_limit_enforcement_scaling_factor`